bulk_course_settings/templates/bulk_course_settings/bulk_settings_list.html

```diff
@@ -63,9 +63,22 @@ <h4>Set Bulk Course settings </h4>
                 <tr>
                     <td>{{ item.get_term_name }}</td>
                     <td>{{ item.get_setting_to_be_modified_display }}</td>
-                    <td>{{ item.get_desired_setting_display }}</td>
+
+                    <td>
+                        {% if item.related_job_id is None %}
+                            {{ item.get_desired_setting_display }}
+                        {% else %}
+                            Reversion
+                        {% endif %}
+                    </td>
                     <td>{{ item.get_workflow_status_display }}</td>
-                    <td><a href="{% url 'bulk_course_settings:revert_setting' %}" class="btn btn-primary">Revert Setting</a></td>
+                    <td>
+                        {% if item.related_job_id is None %}
+                            <a href="{% url 'bulk_course_settings:revert_setting' item.school_id item.id %}" class="btn btn-primary">Revert Setting</a>
```

Comment:
We'll end up moving this button to the details page, but that can be handled as a separate change.